model-optimizer/mo/ops/convolution.py

```diff
@@ -72,6 +71,11 @@ def calc_convolution(input_spatial_shape, stride_spatial_shape, pad_spatial_shap
             Verified to be applicable for both Caffe and ONNX.
         '''
         spatial_val_wo_stride = input_spatial_shape + pad_spatial_shape - kernel_extent
+
+        if np.any(spatial_val_wo_stride < 0):
+            raise Error("Data after padding has dimension less than window size. " +
+                        "Possible reason of error in incorrect input_shape")
```

Comment:
`error in incorrect input_shape` -> `error is incorrectly specified model input shape(s).`

Reply:
fixed